packages/taquito/src/wallet/wallet.ts

```diff
@@ -197,6 +197,10 @@ export class WalletOperationBatch {
 }
 
 export class Wallet {
+  static isWallet(obj: any): obj is Wallet {
```

Comment:
Would also like to understand the change adding static isWallet to Wallet class?